proteus/ctimeIntegration.pyx

```diff
@@ -214,36 +214,30 @@ def applyDGlimitingP1Lagrange1d_withVacuumTol(int enforcePositivity,
                            <double*> Uout.data)
 
 
-def computeElementNeighborShapeGradients(int nElements_global,
-                              int nElementBoundaries_element,
-                              int nSpace,
-                              np.ndarray elementBoundariesArray,
+def computeElementNeighborShapeGradients(np.ndarray elementBoundariesArray,
                               np.ndarray elementNeighborsArray,
                               np.ndarray elementBarycentersArray,
                               np.ndarray elementBoundaryBarycentersArray,
                               np.ndarray elementNeighborShapeGradients):
-    ccomputeElementNeighborShapeGradients(nElements_global,
-                  					  nElementBoundaries_element,
-                  					  nSpace,
+    ccomputeElementNeighborShapeGradients(elementNeighborsArray.shape[0],
```

Comment:
@ejtovar here's an example of where I had to fix the wrappers. Note I just get the integer lengths directly from the required vectors, which simplifies the Python interface and makes it less error prone.